Automattic/wp-calypso
# Remove Popover Cart

#### Changes proposed in this Pull Request

This is a complement to https://github.com/Automattic/wp-calypso/pull/58467; that PR adds a persistent popup cart to the Masterbar; because the cart will now be available on all pages, there is no longer a need for the redundant `PopoverCart` component. This PR removes it.

In addition, this PR updates the e2e tests to use the new cart mechanisms, although those tests are disabled.

Before:
<img width="1088" alt="Screen Shot 2021-11-01 at 7 04 56 PM" src="https://user-images.githubusercontent.com/2036909/139753525-5890b168-fd64-4c83-8443-6f7ff3174f9e.png">

After:
<img width="1087" alt="Screen Shot 2021-11-01 at 7 05 35 PM" src="https://user-images.githubusercontent.com/2036909/139753533-d104a067-719e-48b2-829f-77353505715c.png">


#### Testing instructions

- Visit the `/plans/YOURSITEHERE` page and verify that there is no PopoverCart visible.
